src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Cluster Virtualization Overview
+summary: Learn more about cluster virtualization and how it facilities cluster-to-cluster replication.
+toc: true
+docs_area: deploy
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+
+Refer to the [Known Limitations](#known-limitations) section for further detail.
```

Comment:
I would maybe spell out here that CV is only for PCR in this release rather than just referring to the Known Limitations section. I would still link to the KLs though.

Reply:
That contradicts the earlier feedback from Abbey, Adam, and Steve because it really can't be limited -- you're always going to be using the application virtual cluster as a general-purpose virtual cluster, regardless of whether it is also replicating in the background. This sentence is a stray fragment from applying their feedback. It was felt that it was too strong of a warning-off.